44.md

```diff
@@ -0,0 +1,167 @@
+NIP-44
+======
+
+Encrypted Payloads (Versioned)
+------------------------------
+
+`optional` `author:paulmillr` `author:staab`
+
+The NIP introduces a new data format for keypair-based encryption. This NIP is versioned to allow multiple algorithm choices to exist simultaneously.
+
+The encrypted payload is encoded as base64. The first byte of the payload represents the version. The rest depends on the version.
```

Comment:
base64 without padding?

Reply:
By padding you mean the `===` stuff at the end?

Reply:
I was wondering the same thing. From what I read, base64 padding is to ensure the right number of bytes, extra padding seems unusual.

Reply:
I meant the == stuff. Some base64 impls use it, some don't. Looks like we are using it. Good, that's more normal.

This is very different from padding the plaintext which I talk about elsewhere.